idea/src/org/jetbrains/kotlin/idea/intentions/AddBracesIntention.kt

```diff
@@ -63,6 +64,21 @@ class AddBracesIntention : SelfTargetingIntention<KtElement>(KtElement::class.ja
                 semicolon.delete()
         }
 
+        // Check for single line if expression
+        if (element is KtIfExpression && expression.isInSingleLine() && element.`else` == null) {
+            // Check if a comment is actually underneath (\n) the expression
+            val allElements = element.siblings(withItself = false).filterIsInstance<PsiElement>()
```

Comment:
I think separate `allElements` aren't required, because anyway you perform additional filtering for `sibling`.

Reply:
So, what is your suggestion? that i chain every filtering and avoid the declaration of `allElements`? Don't i need Sequence of PsiElements?